### PR TITLE
feat(dapp): add river points entry

### PR DESCRIPTION
**Pull request by ttillberg**

### CODE REVIEW

File: packages/web3/src/IStaticContractsInfo.ts [hunk 1]
```diff
@@ -9,6 +9,7 @@ export interface BaseChainConfig {
         spaceOwner: Address
         baseRegistry: Address
         riverAirdrop?: Address
+        riverPointsFacet?: Address
```
Comment (texuf) [inline packages/web3/src/IStaticContractsInfo.ts:12]: we don't put facet addresses here. @giuseppecrj or @shuhuiluo  is river points on a new diamond? What goes here?

Comment (ttillberg) [reply packages/web3/src/IStaticContractsInfo.ts:12]: thanks for checking; I'm unfamiliar with this area.

Comment (giuseppecrj) [reply packages/web3/src/IStaticContractsInfo.ts:12]: river points is on riverAirdrop

Comment (texuf) [reply packages/web3/src/IStaticContractsInfo.ts:12]: the riverAirdrop addresses were never put in the right folder. I'm going to re-create my pr that copies everything over and get this fixed up today.

Comment (ttillberg) [reply packages/web3/src/IStaticContractsInfo.ts:12]: Thank you – as far as I understand @texuf is taking a look. I doesn't look like riverAirdrop is actually hooked up to the dApp yet.

**Comment by texuf** [comment]
you can close in favor of https://github.com/river-build/river/pull/1811

**Pull request closed without merging**